tutorials/part2_08_opt/freyberg_opt_2.ipynb

```diff
@@ -740,8 +751,22 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
```

Comment:
@mnfienen did you run that "clear output" script?  Just checking if this metadata should have made it thru...